sklearn/metrics/_classification.py

```diff
@@ -1540,6 +1540,208 @@ def precision_recall_fscore_support(y_true, y_pred, *, beta=1.0, labels=None,
+        Ground truth (correct) target values.
+
+    y_pred : {array-like, label indicator array, sparse matrix} \
+        of shape (n_samples,)
+        Estimated targets as returned by a classifier.
+
+    labels : list, default=None
+        The set of labels to include when ``average != 'binary'``, and their
+        order if ``average is None``. Labels present in the data can be
+        excluded, for example to calculate a multiclass average ignoring a
+        majority negative class, while labels not present in the data will
+        result in 0 components in a macro average. For multilabel targets,
+        labels are column indices. By default, all labels in ``y_true`` and
+        ``y_pred`` are used in sorted order.
+
+    pos_label : str or int, default=1
+        The class to report if ``average='binary'`` and the data is binary.
+        If the data are multiclass or multilabel, this will be ignored;
+        setting ``labels=[pos_label]`` and ``average != 'binary'`` will report
+        scores for that label only.
+
+    average : str, {None, 'binary', 'micro', 'macro', 'samples', 'weighted'}, \
+        default=None
+        If ``None``, the scores for each class are returned. Otherwise, this
+        determines the type of averaging performed on the data:
+
+        ``'binary'``:
+            Only report results for the class specified by ``pos_label``.
+            This is applicable only if targets (``y_{true,pred}``) are binary.
+        ``'micro'``:
+            Calculate metrics globally by counting the total true positives,
+            false negatives and false positives.
+        ``'macro'``:
+            Calculate metrics for each label, and find their unweighted
+            mean.  This does not take label imbalance into account.
+        ``'weighted'``:
+            Calculate metrics for each label, and find their average weighted
+            by support (the number of true instances for each label). This
+            alters 'macro' to account for label imbalance.
+        ``'samples'``:
+            Calculate metrics for each instance, and find their average (only
+            meaningful for multilabel classification where this differs from
+            :func:`accuracy_score`).
+
+    warn_for : tuple or set, for internal use
+        This determines which warnings will be made in the case that this
+        function is being used to return only one of its metrics.
+
+    sample_weight : array-like of shape (n_samples,), default=None
+        Sample weights.
+
+    zero_division : str or int, {'warn', 0, 1}, default="warn"
+        Sets the value to return when there is a zero division:
+           - tpr, fnr: when there are no positive labels
+           - fpr, tnr: when there are no negative labels
+
+        If set to "warn", this acts as 0, but warnings are also raised.
+
+    Returns
+    -------
+    tpr : float (if average is not None), \
+        or ndarray of shape (n_unique_labels,)
```

Comment:
```suggestion
            or ndarray of shape (n_unique_labels,)
        True positive ratio.
```

You can modify the next entries in the same manner.

Reply:
Done.